src/samplers/remote_sampler.js

```diff
@@ -93,11 +101,14 @@ export default class RemoteControlledSampler {
             let maxTracesPerSecond = strategy.rateLimitingSampling.maxTracesPerSecond;
             newSampler = new RateLimitingSampler(maxTracesPerSecond);
         } else {
+            this._metrics.samplerParsingFailure.increment(1);
             this._logger.error('Unrecognized strategy type: ' + JSON.stringify({error: strategy}));
```

Comment:
logtron supports strings, and even if it did support key value pairs the clients don't have to use logtron, and most other loggers support strings.